azure-pipelines.perf-test.yml

```diff
@@ -20,15 +20,10 @@ steps:
       filePath: yarn-ci.sh
     displayName: yarn
 
-  # @fluentui/perf-test needs build and bundle steps
   - script: |
-      yarn lage build --to @fluentui/perf-test --to perf-test --verbose --no-cache --grouped
-    displayName: build  @fluentui v0
-    env:
-      BACKFILL_CACHE_PROVIDER: 'azure-blob'
-      BACKFILL_CACHE_PROVIDER_OPTIONS: '{"connectionString":"$(BACKFILL_CONNECTION_STRING)", "container":"$(BACKFILL_CONTAINER)"}'
```

Comment:
What is the impact of removing the BACKFILL_CACHE_PROVIDER? I searched but I couldn't find any info on this env

Reply:
It's something Ken added but we never turned it on (removed the `--no-cache` option) due to security concerns. So removing the variables is just cleanup.